plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/databinding/DisablableMultiValidator.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.openshift.internal.common.ui.databinding;
+
+import org.eclipse.core.databinding.observable.list.IObservableList;
+import org.eclipse.core.databinding.observable.list.WritableList;
+import org.eclipse.core.databinding.observable.value.IObservableValue;
+import org.eclipse.core.databinding.validation.IValidator;
+import org.eclipse.core.databinding.validation.MultiValidator;
+import org.eclipse.core.databinding.validation.ValidationStatus;
+import org.eclipse.core.runtime.IStatus;
+
+/**
+ * A multi validator that validates a given value observable if validation is
+ * enabled by the given enablement observable. Validation happens based on the
+ * given validation function.
+ * 
+ * @author Andre Dietisheim
+ */
+public class DisablableMultiValidator<T> extends MultiValidator {
```

Comment:
Typo in the className ?

Reply:
@jeffmaury renaming to DisableableMultiValitdator

Reply:
@jeffmaury btw. there are quite a few thing in openshift.commons that we could move to base. But that's for later.